tripal/includes/TripalFields/TripalFieldWidget.inc

```diff
@@ -116,10 +116,18 @@ class TripalFieldWidget {
    * Performs validation of the widget form.
    *
    * Use this validate to ensure that form values are entered correctly.
+   *
    * The 'value' key of this field must be set in the $form_state['values']
    * array anytime data is entered by the user.  It may be the case that there
    * are other fields for helping select a value. In the end those helper
    * fields must be used to set the 'value' field.
+   *
+   * TROUBLESHOOTING:
```

Comment:
I have a slight preference for rolling this into the above explanation.  I think the chado-[tablename]__[columnname] rule s pretty confusing and can't be explains often enough :)

Reply:
I left this as is because I think that the chado-[tablename]__[columnname] rule stands out better in the troubleshooting section and I know it's going to cause people confusion so I want it to stand out as much as possible. ;-p